packages/graphql-language-service-server/src/MessageProcessor.ts

```diff
@@ -1030,7 +1043,7 @@ export class MessageProcessor {
 
     await this._graphQLCache.updateObjectTypeDefinition(
       rootDir,
-      new URL(uri).pathname,
+      fileURLToPath(new URL(uri)),
```

Comment:
it would make more sense to add this in the `GraphQLCache` layer, IMO, since these functions are invoked many more times throughout this file.

Reply:
So you mean passing the uri directly to the cache layer instead of the file Path and resolve the file path there only once?

Reply:
I can try to change that. Whats the fastest way to test if the lang server works?

Reply:
yarn watch and executing the CLI is one way

to test in vscode, just yarn link the server workspace to vscode-graphql repo, and then run the vscode launcher for vscode-grapqhl. and you can run `yarn watch` in the monorepo, though the vscode extension debug launcher still requires refreshes since it compiles at startup time

Reply:
my plan to pull vscode-graphql into this monorepo, and thus a one click DX for the language server for VScode or Vim is a long way off given all the other priorities

Reply:
So I tested it and it works.

Reply:
Whats todo for integration vscode-graphql into this repo? Maybe I can send a pullrequest for that.